src/Admin/SnapshotAdmin.php

```diff
@@ -55,6 +112,7 @@ protected function configureDatagridFilters(DatagridMapper $filter): void
     protected function configureFormFields(FormMapper $form): void
     {
         $form
+            ->add('page')
```

Comment:
This is not ideal, I would like to hide this field... need to think more about this.